README.md

```diff
@@ -100,6 +100,8 @@ Useful articles and learning resources.
 - [Front-end Developer Handbook](https://thoughtworksinc.github.io/front-end-handbook/en/index.html)
 - [Prepare for your next interview](https://github.com/Nabagata/interview-prep)
 - [The Deno Handbook](https://flaviocopes.com/deno/)
+- [Why you should be using JAMSTACK](https://blog.logrocket.com/why-you-should-be-using-jamstack/)
```

Comment:
You might consider adding Logrocket to the **Blogs** section?

Reply:
okay sure